python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py

```diff
@@ -299,6 +299,15 @@ class Meta:
         name = "HookErroredEvent"
 
 
+class GrapheneLogManagerMetadata(graphene.ObjectType):
+    class Meta:
+        name = "LogManagerMetadata"
+
+    logManagerClass = graphene.String(required=True)
```

Comment:
we usually use `NonNull` for fields that shouldn't be nullable - not sure it matters if this is equivalent